grobid-core/src/main/java/org/grobid/core/engines/Engine.java

```diff
@@ -372,8 +372,12 @@ public String processHeader(String inputFile, GrobidAnalysisConfig config, Bibli
             result = new BiblioItem();
         }
 
-        Pair<String, Document> resultTEI = parsers.getHeaderParser().processing2(inputFile, result, config);
-		//Pair<String, Document> resultTEI = parsers.getHeaderParser().processing(inputFile, result, config);
+        Pair<String, Document> resultTEI;
+        if (GrobidProperties.isHeaderUseHeuristics()) {
+            resultTEI = parsers.getHeaderParser().processing2(inputFile, result, config);
+        } else {
+            resultTEI = parsers.getHeaderParser().processing(new File(inputFile), result, config);
+        }
         Document doc = resultTEI.getRight();
```

Comment:
what is the purpose of calling `getRight`? We don't seem to use `doc`.

Reply:
very likely some old stuff not used any more!